### ISSUE TITLE
Doesn't detect logging.Logger arguments `Too many arguments for "Logger"`

### ISSUE THREAD

**Issue by Ran4** [python/mypy#3557]
`logging.Logger` takes a `name` parameter, but mypy doesn't detect it.

#### Example code to reproduce error:

```python
    #!/usr/bin/env python3
    import logging
    logging.Logger("Logger name")
    logging.Logger(name="Another logger name")
```

#### Output of mypy:

```
    $ mypy .
    mypy_logging_test.py:3: error: Too many arguments for "Logger"
    mypy_logging_test.py:4: error: Unexpected keyword argument "name" for "Logger"
```

Using Python 3.5.2, mypy==0.511 on Ubuntu 16.04

**Comment by ilevkivskyi** [comment]
Thanks for reporting! It looks like a typeshed issue (missing ``Logger.__init__``), could you please report it in [typeshed tracker](https://github.com/python/typeshed/issues)?
